modules/olm-building-operator-catalog-image.adoc

```diff
@@ -73,7 +77,8 @@ $ oc adm catalog build \
     --filter-by-os="linux/amd64" \//<3>
     --to=<registry_host_name>:<port>/olm/redhat-operators:v1 \//<4>
     [-a ${REG_CREDS}] \//<5>
-    [--insecure] <6>
+    [--insecure] \//<6>
+    [--auth-token <password>] <7>
```

Comment:
Should we specify how to get the auth token? It's not very obvious how to, though I suppose if you're using this flag you would already be familiar with using app-registry?

Reply:
I put a link to the quay.io page up in the Prerequisites so that it didn't clutter up the steps.

Reply:
Unless that's not the right token to grab.